### DISCUSSION TITLE
2005/07/09/%E5%88%A9%E7%94%A8labview%E5%B7%A5%E7%A8%8B%E5%BA%93%E5%AE%9E%E7%8E%B0%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%BC%96%E7%A8%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#39] [Announcements]
# 2005/07/09/%E5%88%A9%E7%94%A8labview%E5%B7%A5%E7%A8%8B%E5%BA%93%E5%AE%9E%E7%8E%B0%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%BC%96%E7%A8%8B

阮奇桢

https://blog.qizhen.xyz/2005/07/09/%E5%88%A9%E7%94%A8labview%E5%B7%A5%E7%A8%8B%E5%BA%93%E5%AE%9E%E7%8E%B0%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%BC%96%E7%A8%8B

**Comment by ruanqizhen** [comment]
Qizhen
2010年02月1日 09:50 编辑
有一些视频，你可以参考一下：http://www.gsdzone.net/Video.aspx?cid=43&vid=78

回复
Unknown
2010年01月31日 21:44 编辑
阮老师，能不能多讲解些关于面向对象编程，看了这似懂非懂。

回复
刘伟印
2008年08月15日 11:38 编辑
原来以为工程库与传统的LabVIEW的LLB文件是一个东西，现在才知道区别很大，谢谢！

回复
周捷
2007年01月29日 13:38 编辑
确实是蛮有收获的每次浏览阮大侠的blog,谢谢!

回复
鑫
2007年01月11日 09:09 编辑
生成类的实例(new)怎么还要用到C语言,太麻烦了吧

回复
Qizhen
2006年09月7日 23:00 编辑
只要不是工作特别忙，我都会继续写下去的。

回复
清扬
2006年09月7日 20:14 编辑
lv8.2新增的OO编程还不是很了解,希望你能做一次详细的专题.
每次来这看你的blog都会有所收获,一定要坚持写下去啊~

回复
Qizhen
2006年09月3日 20:21 编辑
我写这篇文章的时候，LabVIEW 8.2 还没有出来。现在 LabVIEW 8.2 本身就以支持面向对象的编程方法，所以这里介绍的方法有点过时。我有时间会再写一篇关于新 LVOOP 的文章。

回复
._小茶ˇ┆
2005年07月11日 23:36 编辑
好漂亮的PIC..喜欢..

回复
